src/picongpu/CMakeLists.txt

```diff
@@ -219,73 +93,6 @@ find_package(Boost 1.57.0 REQUIRED COMPONENTS program_options regex filesystem
 include_directories(SYSTEM ${Boost_INCLUDE_DIRS})
 set(LIBS ${LIBS} ${Boost_LIBRARIES})
 
-# Boost 1.55 added support for a define that makes result_of look for
-# the result<> template and falls back to decltype if none is found. This is
-# great for the transition from the "wrong" usage to the "correct" one as
-# both can be used. But:
-# 1) Cannot be used in 7.0 due to nvcc bug:
-#    http://stackoverflow.com/questions/31940457/
-# 2) Requires C++11 enabled as there is no further check in boost besides
-#    the version check of nvcc
-if( (NOT CUDA_VERSION VERSION_EQUAL 7.0) AND (CMAKE_CXX_STANDARD EQUAL 11) )
-    add_definitions(-DBOOST_RESULT_OF_USE_TR1_WITH_DECLTYPE_FALLBACK)
-else()
-    # Fallback
-    add_definitions(-DBOOST_RESULT_OF_USE_TR1)
-endif()
-
-
-# Boost >= 1.60.0 and CUDA != 7.5 failed when used with C++11
-# seen with boost 1.60.0 - 1.62.0 (atm latest) and CUDA 7.0, 8.0 (atm latest)
-# CUDA 7.5 works without a workaround
-if( (Boost_VERSION GREATER 105999) AND
-    (NOT CUDA_VERSION VERSION_EQUAL 7.5) )
-    # Boost Bug https://svn.boost.org/trac/boost/ticket/11897
-    message(STATUS "Boost: Disable template aliases")
-    add_definitions(-DBOOST_NO_CXX11_TEMPLATE_ALIASES)
-endif()
-
-
-################################################################################
-# Find OpenMP
```

Comment:
We **use** `OpenMP` in PIConGPU directly, so PIConGPU **should** run a `find_package` for `OpenMP` itself (again)!